test/functionalTest/cases/2092_incorrect_url_bug/check_incorrect_urls.test

```diff
@@ -0,0 +1,161 @@
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Context subscripion description field
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0-255
+
+--SHELL--
+
+#
+# 01. Create subscription without url // and get error
+# 02. Create subscription without url /// and get error
+# 03. Create subscription without url http:///a and get error
+#
+
+echo "01. Create subscription without url // and get error"
+echo "===================================================="
+payload='
+{
+    "subject": {
+        "entities": [
+            {
+                "type": "Room"
+            }
+        ],
+        "condition": {
+            "attrs": [ "temperature" ]
+         }
+    },
+    "notification": {
+        "http": {
+            "url": "//"
+        }
+    }
+}
+'
+orionCurl --url /v2/subscriptions --payload "$payload" --json
+echo
+echo
+
+
+echo "02. Create subscription without url /// and get error"
+echo "====================================================="
+payload='
+{
+    "subject": {
+        "entities": [
+            {
+                "type": "Room"
+            }
+        ],
+        "condition": {
+            "attrs": [ "temperature" ]
+         }
+    },
+    "notification": {
+        "http": {
+            "url": "///"
+        }
+    }
+}
+'
+orionCurl --url /v2/subscriptions --payload "$payload" --json
+echo
+echo
+
+
+echo "03. Create subscription without url http:///a and get error"
+echo "==========================================================="
+payload='
+{
+    "subject": {
+        "entities": [
+            {
+                "type": "Room"
+            }
+        ],
+        "condition": {
+            "attrs": [ "temperature" ]
+         }
+    },
+    "notification": {
+        "http": {
+            "url": "http:///a"
+        }
+    }
+}
+'
+orionCurl --url /v2/subscriptions --payload "$payload" --json
+echo
+echo
+
+
+--REGEXPECT--
+01. Create subscription without url // and get error
+====================================================
+HTTP/1.1 400 Bad Request
+Content-Length: 50
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "Invalid URL",
```

Comment:
I find this description a little dry ...

Invalid URL for notifications ?

Otherwise you could understand that the broker is complaining about the URL of the subscription request (which would probably be a 404 not found, but still)